### ISSUE TITLE
native keyboard features

### ISSUE THREAD

**Issue by edvenkat** [EddyVerbruggen/cordova-plugin-native-keyboard#41]
hai, it is really very useful. Thanks for this plugin.

We have some doubt please help me
In demo/index_all_features.html file we are added the configureNativeKeyboard how we add the "keepOpenAfterSubmit" and will / did keyboard show / hide function. i try to add but it is not working please advice.......

**Comment by EddyVerbruggen** [comment]
This is a bit confusing to me, can you throw your project on a repo so I can take a look?

**Comment by edvenkat** [comment]
Consider the form with following fields "first name", "Last name","email" etc.... 
i have changed the retuntype to "Next".user click the next button i focus the next input field... it is working but the Keyboard hide and again show..... i added the "keepOpenAfterSubmit" to true i am getting same result please advice........

**Comment by EddyVerbruggen** [comment]
I think you're not using the messenger part of this plugin, but one of the "hidden" features. I will really need to see your code instead of a description of it.

**Comment by edvenkat** [comment]
Ya i am using data-nativekeyboard like this.. i refereed in demo/index_all_features.html file..
<input data-nativekeyboard='configureNativeKeyboard("#FF0000")' type="tel" value="+31650123456" />
FYI - i don't have code now. please advice...

**Comment by EddyVerbruggen** [comment]
I will not be able to help unless you share a repo reproducing this issue. Please reopen when you have one.

**Comment by edvenkat** [comment]
could you please refer this page https://github.com/edvenkat/myapp/blob/master/www/index.html

**Comment by edvenkat** [comment]
@EddyVerbruggen understand?

**Comment by EddyVerbruggen** [comment]
I kinda do, but you're mixing up a few things: `keepOpenAfterSubmit` is ONLY intended to be used with the `showMessenger` function, not with anything else.

If the user taps inside other input fields then the keyboard will remain open, but if it's triggered by a Next button that sends control back to JavaScript before jumping to the next field, then I'm sure it will hide and show the keyboard. That's not something I've added support for in the plugin yet as that's not what is officially released (only the messenger is).

That being said I like you case and perhaps adding `keepOpenAfterSubmit` will enable this kind of usage for you.

I'll reopen and perhaps in the future this will be supported.

**Comment by edvenkat** [comment]
When will you complete this... i need urgently.could you please tell some idea where i update? please

**Comment by EddyVerbruggen** [comment]
Not soon.

**Comment by edvenkat** [comment]
can you tell where i update code for ios? and can you add following function also shrinkView and hideFormAccessoryBar, keyboardDidShow etc

**Comment by EddyVerbruggen** [comment]
No. This is exactly why this is a paid plugin and why I ONLY currently support the stuff that's in the readme (the messenger feature). Don't try and dictate what I need to do and when I need to do it.

**Comment by edvenkat** [comment]
Okay.....do u know cordova-plugin-keyboard? how to add the caret(cursor) color please advice

**Comment by EddyVerbruggen** [comment]
That's impossible with Cordova because the caret is rendered by the system and the textfield is a browser widget. You can't style it with CSS, so that plugin has no control over the caret color. This plugin uses a different approach to rendering input fields making it possible to use an alternate caret color.

**Comment by edvenkat** [comment]
Okay.please update soon........... how to add / customize the  accessory bar for drop down
